content/resources/csc236.md

```diff
@@ -0,0 +1,48 @@
+---
+title: CSC236
+icon: /icons/csc236.svg
+desc: Introduction to the Theory of Computation
+link: csc236
+tags:
+  - utm-cs
+---
+
+# CSC236: Introduction to the Theory of Computation
+#### Prerequisite: [CSC148](./csc148), [MAT102](./mat102)
+
+<grid-1-x-2 title="Fall 2020 Class Website" img-src="https://i.imgur.com/MX1GL74.png" link="https://mcs.utm.utoronto.ca/~236/" desc="All credits to Ilir Dema" button="Check it out!"></grid-1-x-2>
+
+### What is CSC236?
+CSC236 is a course that focuses on proving the correctness and the runtime of both iterative and recursive algorithms unlike the other second year computer science courses.
```

Comment:
I don't think it's necessary to put "unlike the other second year computer science courses"